packages/features/embed/Embed.tsx

```diff
@@ -286,12 +304,23 @@ const EmailEmbed = ({
         <Collapsible open>
           <CollapsibleContent>
             <div className="text-default mb-[9px] text-sm">{t("duration")}</div>
-            <TextField
-              disabled
-              label={t("duration")}
-              defaultValue={eventType?.length ?? 15}
-              addOnSuffix={<>{t("minutes")}</>}
-            />
+            {durationsOptions.length > 0 ? (
+              <Select<{ label: string; value: number }>
+                value={durationsOptions.find((option) => option.value === selectedDuration)}
+                options={durationsOptions}
+                onChange={(option) => {
+                  setSelectedDuration(option?.value);
+                  setSelectedDatesAndTimes({});
+                }}
+              />
+            ) : (
+              <TextField
+                disabled
+                label={t("duration")}
+                defaultValue={eventType?.length ?? 15}
+                addOnSuffix={<>{t("minutes")}</>}
+              />
```

Comment:
Conditionally rendering Select in case eventType has multiple duration. Can be changed to only show Select and disable it in case only a single duration. Lmk what you think